### ISSUE TITLE
It seems that output .prj always contains the same wkt projection string

### ISSUE THREAD

**Issue by ellvtr** [mbloch/mapshaper#430]
**Output .prj always contains the same wkt projection string**

This command:
```
mapshaper.cmd -i cal-3-features-simplified.json -o output/export.file format=shapefile encoding=utf-8 
```
Results in this .prj file:
```
GEOGCS["WGS84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["degree",0.017453292519943295]]
```
Even though the source is in NAD27/EPSG:4267, see attached geojson. No re-reprojection is required, just conversion from geojson to shape format.

When I add various supported `-proj` parameters to the command, it has no effect on the output .prj file, it outputs the same string each time. Tried some at random:
```
mapshaper.cmd -i cal-3-features-simplified.json -o output/export.file format=shapefile encoding=utf-8 -proj EPSG:2831
... -proj wag1
... -proj utm
... denoy, eck1 and others ...
```

**Expected behaviour:**
That the .prj file the projection that matches the source when no re-projection is required.

[cal-3-features-simplified.json.txt](https://github.com/mbloch/mapshaper/files/5061525/cal-3-features-simplified.json.txt)

**Comment by mbloch** [comment]
Mapshaper is unable to import the "crs" property from GeoJSON files. 

The current revision of the GeoJSON spec (https://tools.ietf.org/html/rfc7946) has removed  the "crs" property, and states that GeoJSON files should only use WGS84.

That said, I believe that other open source software does support the older form of GeoJSON that uses "crs". QGIS and OGR are examples. Can you use ogr2ogr to convert your NAD27 GeoJSON files?

I don't plan to add support for GeoJSON CRSs, but I think mapshaper should at least print a useful warning message when it loads a GeoJSON file containing a "crs" property.

Thanks for filing this issue, I'm happy to discuss this further if you'd like.

**Comment by ellvtr** [comment]
Thanks. We're currently using MapShaper in the browser/frontend, so ogr2ogr and QGIS are not alternatives. The user will download the geosjon in the browser memory to a shape-file which is created on-the-fly by MapShaper.

So I know what the CRS is, when I start the conversion from json to shape-file. In this case the incomping json is EPSG:4267 and the output shp should be the same. 

What would be the correct command for that?

(I realize that MapShaper may not support NAD27/EPSG:4267 but no re-projection is required).

**Comment by ellvtr** [comment]
Odd, with EPSG:2193 (attached json), no output .prj file is created with this command:
```
mapshaper.cmd -i RB_Extent_Poly-epsg2193.json -o output/export.* format=shapefile encoding=utf-8
```

[RB_Extent_Poly-epsg2193.json.txt](https://github.com/mbloch/mapshaper/files/5062565/RB_Extent_Poly-epsg2193.json.txt)

**Comment by mbloch** [comment]
One problem with your example above is that the `-o` command needs to go at the very end. Mapshaper runs commands in sequence from left to right.  So in your example, mapshaper was exporting your data before it ran the `-proj` command. There was no `.prj` because mapshaper could tell that the file contained projected coordinates (they were outside of the lat-long range) but didn't know which projection to use.

Here's a corrected command, which generates a `.prj` correctly:
```
mapshaper RB_Extent_Poly-epsg2193.json -proj from='ESPG:2193' -o export.shp
```

Note that assigning a CRS like this will not work with a NAD27 dataset, because mapshaper throws an error when it sees `NAD27/EPSG:4267`. I can work on removing this error, when I have the time.

**Comment by ellvtr** [comment]
Thanks a bundle, I appreciate your help! Is there a way to make sure that coordniates are not re-calculated but simply passed on to the new format?

Perhaps by making it explicit like this?
```
mapshaper RB_Extent_Poly-epsg2193.json -proj from='ESPG:2193' to='ESPG:2193' -o export.shp
```

**Comment by mbloch** [comment]
The `from=` option assigns a CRS to the dataset, and if you don't specify a destination CRS, no coordinate calculations occur.

I can see that the term `from` is misleading, since it implies a transformation. I should deprecate that name and pick a new name... would `init` be a better name for an option that assigns a CRS without changing coordinates?

**Comment by ellvtr** [comment]
I believe it might, thanks

**Comment by ellvtr** [comment]
Following your latest feedback and my testing, it turns out that I can convert data in any projection to shapefile without reprojecting the coordniates e.g. like this:
```
mapshaper -i cal-3-features-simplified.json encoding=utf-8 -o output/export.json format=shapefile
```
The output sometimes has a .prj file that says WGS84 but that can be ignored. I will make sure my script removes the .prj to avoid misunderstandings.

Is that correct?

**Comment by ellvtr** [comment]
By the way, the coordinates appear to be rounded off in some cases in spite of them not being re-projected. The result in this image:
![2020-08-17 11_34_37-output_export json ↔ cal-3-features-simplified json - NAD27-mapshaper - Visual S](https://user-images.githubusercontent.com/12038749/90383971-2be43300-e081-11ea-9999-4ff0a5712c6f.png)

Is a result of this command:
```
mapshaper -i cal-3-features-simplified.json encoding=utf-8 -o output/export.json format=shapefile
```

**Comment by mbloch** [comment]
When data is imported, mapshaper converts it into an internal data format. Some information in the original GeoJSON is lost -- for example, whether the feature was a Polygon or a MultiPolygon with a single ring. When it comes time to export, mapshaper uses the Polygon type for single-ring polygons and MultiPolygon for multi-ring polygons.

What looks like rounding is due to the fact that the original coordinates contained more digits than can be distinguished using 64-bit floating point arithmetic. If you paste the original numeric strings into a browser console or Node.js console, you will see that the last few digits disappear:

![image](https://user-images.githubusercontent.com/1232222/90391570-5a4f1780-e05b-11ea-9b59-8009708e23f6.png)

**Comment by ellvtr** [comment]
Thanks,

And is it correct, that I can convert data in any projection to shapefile without reprojecting the coordniates?
I can disregard the .prj file and rest assured that the output coordinates are the same as the input coordinates?
```
mapshaper -i cal-3-features-simplified.json encoding=utf-8 -o output/export.* format=shapefile
mapshaper -i RB_Extent_Poly-epsg2193.json encoding=utf-8  -o output/export.* format=shapefile
```

**Comment by mbloch** [comment]
You are correct, mapshaper will not change coordinates unless you use the `-proj` command to reproject your dataset.

**Comment by ellvtr** [comment]
Awesome, thanks!
